packages/platform/atoms/availability/AvailabilitySettings.tsx

```diff
@@ -76,6 +76,13 @@ export type CustomClassNames = {
   formClassName?: string;
   timezoneSelectClassName?: string;
   subtitlesClassName?: string;
+  scheduleClassName?: {
```

Comment:
I think we can call this scheduleClassnames with an S at this end to make it clear it contains multiple one and is not just a string like the rest